demo/pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[tool.hatch.version]
+path = "src/__init__.py"
+allow-direct-references = true
+
+[project]
+name = 'demo'
+description = 'FastUI demo project'
+authors = [{ name = 'Samuel Colvin', email = 's@muelcolvin.com' }]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: Implementation :: CPython',
+    'Programming Language :: Python :: Implementation :: PyPy',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3 :: Only',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: System Administrators',
+    'Operating System :: Unix',
+    'Operating System :: POSIX :: Linux',
+    'Environment :: Console',
+    'Environment :: MacOS X',
+    'Framework :: Hypothesis',
+    'Framework :: Pydantic',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    'Topic :: Internet',
+]
+requires-python = '>=3.8'
+dependencies = [
+    "fastapi>=0.110.2",
+    "httpx>=0.27.0",
+    "openai==1.10.0",
+    "PYjwt>=2.8.0",
+    "pydantic[email]>=2.7.1",
```

Comment:
Are these all truly dependencies?

Reply:
that's not the pyproject for fastui, that's for the demo.
Yes they're all. but it shouldn't affect anyway, because demo itself is "dev" package right? it's not that we ship it to client.